src/citrine/resources/descriptors.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, project_id: UUID, session: Session):
         self.project_id = project_id
         self.session: Session = session
 
-    def from_predictor_responses(self, predictor: Predictor,
+    def from_predictor_responses(self, predictor: Predictor, *,
```

Comment:
Both the predictor and the inputs are equally important parts of the function call, so I think they should both require keywords

Reply:
That makes sense. I was thinking that because predictor was in the method name, it would be redundant, but I think I'd prefer to err on the side of enforcing kwargs.
I'll also add it to https://github.com/CitrineInformatics/citrine-python/pull/628/files#diff-5fb493c283c9b66db198bd69f057b8b7bef132929d0243d8e2a5be268f6da418R46